wordpress-mobile/WordPress-Android
# Activity type filter UI

Parent issue #13268 

I applied the styles per the design comments and some things don't match (text color, line height, divider color , ...).

What color do you mean by "high emphasis" - could you please share the hex code so I can check if we have it defined in the code. Thanks!

<img width="500" alt="Screenshot 2020-12-08 at 16 46 39" src="https://user-images.githubusercontent.com/2261188/101506006-39496400-3975-11eb-86c8-68ba5605e223.png">

This PR polishes the UI of the activity type filter screen

To test:
1. My site
2. Activity Log
3. Activity Type filter
4. Make sure the screen looks per the designs (I ignored the loading screen for now)

PR submission checklist:
